docs/tutorials/linear/700_progot.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>i would replace the "(dual of the entropic)" mention, as ProgOT doesn't return currently dual solutions, and, to some extent, is not really the EOT problem either.</p><p> </p><p>For instance:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">&nbsp;ProgOT is a progressive algorithm that outputs approximate solutions to the OT problem, returning both, a coupling matrix ... that aligns the source and target datasets (primal OT problem), and a map estimator (valid out of sample) ... that pushes-forward efficiently points from the source to the target.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/567/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done, thanks!